google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -3272,7 +3271,7 @@ public void testExecuteSelectWithCredentials() throws SQLException {
 
     ConnectionSettings connectionSettings =
         ConnectionSettings.newBuilder()
-            .setJobTimeoutMs(10L) //  Force non-fast query to use BigQueryReadClient.
+            .setPriority(Priority.INTERACTIVE) //  Force non-fast query to use BigQueryReadClient.
```

Comment:
How/why would this force read client?  Interactive is the default priority already.

Reply:
We check a few criteria in the ConnectionSettings to determine if fast query should be used in:
https://github.com/googleapis/java-bigquery/blob/8f2e5c542760ecd7c217c36c80cb3b5aebee6a73/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ConnectionImpl.java#L1209
This forces the check to fail and executeSelect to use the non fast query path:
https://github.com/googleapis/java-bigquery/blob/8f2e5c542760ecd7c217c36c80cb3b5aebee6a73/google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/ConnectionImpl.java#L245C1-L246C1

Reply:
Thanks for the clarification!